test/answering_machines.uts

```diff
@@ -147,19 +147,18 @@ test_am(LLMNR_am,
 
 = mDNS_am
 def check_mDNS_am_reply(packet):
-    assert packet[Ether].src == get_if_hwaddr(conf.iface)
     assert packet[Ether].dst == "01:00:5e:00:00:fb"
-    assert packet[IP].src == get_if_addr(conf.iface)
     assert packet[IP].dst == "224.0.0.251"
+    assert packet[IP].ttl == 1, "a"
     assert packet[UDP].dport == 5353
     assert packet[UDP].sport == 5353
     assert DNS in packet and packet[DNS].ancount == 1 and packet[DNS].qdcount == 0
     assert packet[DNS].an[0].rdata == "192.168.1.1"
     assert packet[DNS].an[0].rrname == b"TEST.local."
-    assert packet[DNS].an[0].ttl == 10
+    assert packet[DNS].an[0].ttl == 60
```

Comment:
I think the recommended ttl value for A RRs is 120: https://datatracker.ietf.org/doc/html/rfc6762#section-10 (unless replies are sent in response to legacy queries. In that case TTLs should not be greater than ten seconds: https://datatracker.ietf.org/doc/html/rfc6762#section-6.7).

Reply:
You're very right.

For compatibility we should probably leave 10. I'll do that and document it.

Reply:
> For compatibility we should probably leave 10

Agreed. Though one downside is that short TTLs make some mDNS daemons (like systemd-resolved) go crazy when they conflict with whatever has already been in its cache  :-) I'm not complaining though. It helped to cover one less traveled code path there. I had a weird test script for that and now I can just launch `mdnsd`.

Reply:
Done.